config-overrides.js

```diff
@@ -25,11 +25,11 @@ const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
 
 module.exports = function override(config, env) {
   config.resolve = {
+    ...config.resolve,
     fallback: {
+      ...config.resolve.fallback,
       stream: require.resolve('stream-browserify'),
     },
-    // NOTE: I added this; missing in Substrate template and causes missing module errors;
-    extensions: [...config.resolve.extensions, '.ts', '.js'],
```

Comment:
You need first to destruct the existing "resolve" properties before adding the fallback. That way you don't need to add "extensions".